components/brave_wallet/browser/BUILD.gn

```diff
@@ -590,6 +610,216 @@ generated_types("meld_integration_responses") {
+    "zcash/zcash_grpc_utils_unittest.cc",
+    "zcash/zcash_keyring_unittest.cc",
+    "zcash/zcash_resolve_transaction_status_task_unittest.cc",
+    "zcash/zcash_serializer_unittest.cc",
+    "zcash/zcash_transaction_unittest.cc",
+    "zcash/zcash_transaction_utils_unittest.cc",
+    "zcash/zcash_wallet_service_unittest.cc",
+  ]
+
+  deps = [
+    ":browser",
+    ":constants",
+    ":generated_json_rpc_responses",
+    ":generated_swap_responses",
+    ":hd_keyring",
+    ":pref_names",
+    ":test_data",
+    ":test_support",
+    ":transaction",
+    ":utils",
+    "//base/test:test_support",
+    "//brave/components/brave_component_updater/browser",
+    "//brave/components/brave_component_updater/browser:test_support",
+    "//brave/components/brave_wallet/browser/bitcoin:generated_bitcoin_rpc_responses",
+    "//brave/components/brave_wallet/browser/cardano:generated_cardano_rpc_blockfrost_api",
+    "//brave/components/brave_wallet/browser/internal:hd_key",
+    "//brave/components/brave_wallet/common",
+    "//brave/components/brave_wallet/common:buildflags",
+    "//brave/components/brave_wallet/common:common_constants",
+    "//brave/components/brave_wallet/common:mojom",
+    "//brave/components/brave_wallet/common:rust_lib",
+    "//brave/components/brave_wallet/common:test_support",
+    "//brave/components/constants",
+    "//brave/components/decentralized_dns/core",
+    "//brave/components/ipfs",
+    "//brave/components/json",
+    "//brave/components/resources:strings_grit",
+    "//brave/components/services/brave_wallet/public/cpp/utils",
+    "//brave/components/services/brave_wallet/public/mojom",
+    "//components/component_updater:test_support",
+    "//components/prefs",
+    "//components/prefs:test_support",
+    "//components/sync_preferences:test_support",
+    "//components/update_client",
+    "//net:test_support",
+    "//services/data_decoder/public/cpp:test_support",
+    "//services/network:test_support",
+    "//testing/gtest",
+    "//ui/base:base",
+    "//url",
+  ]
+
+  if (enable_orchard) {
+    sources += [
+      "internal/hd_key_zip32_unittest.cc",
+      "internal/orchard_block_scanner_unittest.cc",
+      "internal/orchard_bundle_manager_unittest.cc",
+      "internal/orchard_storage/orchard_storage_unittest.cc",
+      "internal/orchard_sync_state_unittest.cc",
+      "zcash/zcash_auto_sync_manager_unittest.cc",
+      "zcash/zcash_blocks_batch_scan_task_unittest.cc",
+      "zcash/zcash_get_chaintip_status_task_unittest.cc",
+      "zcash/zcash_scan_blocks_task_unittest.cc",
+      "zcash/zcash_shield_sync_service_unittest.cc",
+      "zcash/zcash_verify_chain_state_task_unittest.cc",
+    ]
+    deps += [
+      "//brave/components/brave_wallet/browser/internal:orchard_bundle",
+      "//brave/components/brave_wallet/browser/internal:test_support",
+      "//brave/components/brave_wallet/browser/internal/orchard_storage",
+    ]
+  }
+
+  if (!is_ios) {
```

Comment:
you can't just suppress these deps in ios unless they have IS_IOS guards around them in the tests